signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/OrganizerSignupMBean.java

```diff
@@ -489,6 +503,15 @@ public void loadAllAttendees(SignupMeeting meeting) {
 			return;
 		}
 
+		if (!allSignupUsers.isEmpty()) {
+			Collections.sort(allSignupUsers, new Comparator<SignupUser>() {
+				public int compare(SignupUser a1, SignupUser a2) {
+					UserSortNameComparator userComparator = new UserSortNameComparator();
+					return userComparator.compare(getUser(a1.getInternalUserId()), getUser(a2.getInternalUserId()));
```

Comment:
You could make getUser non static here and use OrganizerSignupMBean.this.getUser.

Reply:
Also what happens to SignupUser.compareTo ? That should be fixed and that should be the only place the sorting is done. It doesn't make sense to add sorting logic to just the OrganizerSignupMBean.